src/templates/build-page/taco/TacoExamples.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { FC, useState } from "react"
+import { Box, Stack, Collapse, Flex } from "@chakra-ui/react"
+import { ResponsiveStack } from "../../../components/PageSection"
+import { Image } from "../../../components/Image"
+import { SectionInfo } from "../../../types"
+import Card from "../../../components/Card"
+
+export interface TacoExamplesProps {
+  tacoExamples: SectionInfo[]
+}
+
+export interface TacoExampleProps extends SectionInfo {
+  togglePanel: (index: number) => void
+  expandedPanel: number | null
+  index: number
+}
+
+export const TacoExampleCard: FC<TacoExampleProps> = ({
+  togglePanel,
+  expandedPanel,
+  index,
+  title,
+  description,
+}) => {
+  return (
+    <Box
+      boxShadow="0px 0px 90px 4px rgba(153, 116, 255, 0.04)"
+      rounded="2xl"
+      p="2px"
+      bgGradient={
+        expandedPanel === index
+          ? "linear-gradient(120.19deg, #BD30FF 3.32%, #7D00FF 95.02%)"
+          : "none"
+      }
+    >
+      <Card w={{ base: "100%", md: "420px" }}>
+        <Stack
+          direction="row"
+          alignItems="center"
+          justifyContent="space-between"
+        >
+          <Card.Title fontWeight="medium">{title}</Card.Title>
+          <Box
+            fontSize="28px"
+            cursor="pointer"
+            onClick={() => togglePanel(index)}
+          >
+            {expandedPanel === index ? "-" : "+"}
```

Comment:
Imo would be much more intuitive to be able to click anywhere on the unexpanded panel to open or close it, not just the **+** or **-** 

![image](https://github.com/threshold-network/website/assets/2332199/99bd318a-4426-45b9-8529-04e790c95ccc)

Reply:
Also it would be better if the default webpage had the NFT-gating box already expanded so we could see the text (_Predicate decryption rights on the holding of special-purpose NFTs. Works for digital media, event streams and beyond_), since we see the associated images by default

Reply:
when you open the webpage, this box should be expanded and highlighted purple by default. then people will click the other two boxes (Secret recovery and Inference protection)